src/IntervalFormatter.test.js

```diff
@@ -83,9 +83,13 @@ describe('IntervalFormatter', () => {
       .returns(unsubscriber)
```

Comment:
I think it worths to test with the [sinon's faker timer](http://sinonjs.org/releases/v1.17.7/fake-timers/) if the time has advanced automatically